android/src/app/organicmaps/editor/AdvancedTimetableFragment.java

```diff
@@ -65,6 +68,7 @@ private void initViews(View view)
     mExamplesTitle = view.findViewById(R.id.tv__examples_title);
     setExampleDrawables(R.drawable.ic_type_text, R.drawable.ic_expand_more);
     setTextChangedListener(mInput, mListener);
+    mSaveButton = getParentFragment().getParentFragment().getView().findViewById(R.id.save);
```

Comment:
Should this variable be defined/retrieved locally in the function and passed to the lambda instead of storing it as a class member?

Reply:
If we move it to onTextChanged wouldn't it make the code slower? 
Because each time the text would change it would call the above line.

Reply:
I did not write to move it into onTextChanged. I wrote that it can be defined in the function (`setTextChangedListener`) and then passed into the onTextChanged. The variable is not needed outside of the lambda scope.

Reply:
setTextChangedListener() cannot be referenced from a static context so we can move it into the setTextChangedListener()  function but then we would need to make it non-static.